src/search/query.rs

```diff
@@ -55,6 +56,14 @@ impl Query {
         self
     }
 
+    /// *Adds* an `assignee` statement to the query.
+    ///
+    /// Results in `assignee:username`.
+    pub fn assignee(mut self, assignee: &str) -> Self {
```

Comment:
Could you make the function parameters match the documentation? i.e. `pub fn assignee(mut self, username: &str)`. While unlikely, there's a chance that someone could interpret the value they pass for assignee to be *before* the `:`, not after.

Reply:
Yeah, let's do that.